### PR TITLE
[improvement](spill) improve spill log printing

### PR DESCRIPTION
**Pull request by jacktengg**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by jacktengg** [comment]
run buildall

**Review by yiguolei — DISMISSED** [review]

**Comment by jacktengg** [comment]
run external

File: be/src/pipeline/pipeline_task.cpp [hunk 1]
```diff
@@ -315,7 +315,8 @@ bool PipelineTask::should_revoke_memory(RuntimeState* state, int64_t revocable_m
     wg->check_mem_used(&is_wg_mem_low_water_mark, &is_wg_mem_high_water_mark);
     if (is_wg_mem_high_water_mark) {
         if (revocable_mem_bytes > min_revocable_mem_bytes) {
-            LOG_EVERY_N(INFO, 10) << "revoke memory, hight water mark";
+            LOG_EVERY_T(INFO, 1) << "query " << print_id(state->query_id())
```
Comment (yiguolei) [inline be/src/pipeline/pipeline_task.cpp:318]: Not print this log, it should be a debug log. 
This method is called by pipeline engine during every loop.
If there is many queries, the log is too often.

File: be/src/runtime/workload_group/workload_group_manager.cpp [hunk 2]
```diff
@@ -201,7 +201,7 @@ void WorkloadGroupMgr::refresh_wg_memory_info() {
                 PrettyPrinter::print(process_mem_used, TUnit::BYTES),
                 PrettyPrinter::print(sys_mem_available, TUnit::BYTES),
                 PrettyPrinter::print(all_queries_mem_used, TUnit::BYTES));
-        VLOG_EVERY_N(1, 10) << debug_msg;
+        LOG_EVERY_T(INFO, 1) << debug_msg;
```
Comment (yiguolei) [inline be/src/runtime/workload_group/workload_group_manager.cpp:204]: very 10 seconds

**Comment by jacktengg** [comment]
run buildall

**Review by mrhhsg — APPROVED** [review]
LGTM

**Review by HappenLee — APPROVED** [review]
LGTM

**Pull request merged**